src/client/datascience/constants.ts

```diff
@@ -165,7 +165,7 @@ export enum Telemetry {
     CopySourceCode = 'DATASCIENCE.COPY_SOURCE',
     RestartKernel = 'DS_INTERNAL.RESTART_KERNEL',
     RestartKernelCommand = 'DATASCIENCE.RESTART_KERNEL_COMMAND',
-    ExportNotebook = 'DATASCIENCE.EXPORT_NOTEBOOK',
+    ExportNotebookInteractive = 'DATASCIENCE.EXPORT_NOTEBOOK_INTERACTIVE',
```

Comment:
```suggestion
    ExportNotebookInteractive = 'DATASCIENCE. EXPORT_NOTEBOOK',
```

Please do not change existing names, that still mess up with the existing data.
We can change the name of the constant, but not the underlying value.